travis-scripts/before_install_osx.sh

```diff
@@ -37,7 +37,8 @@ brew update
 brew config
 
 sed -i -e "s|{TRAVIS_PULL_REQUEST}|${TRAVIS_PULL_REQUEST}|g" ./travis-scripts/fontforge.rb
-sudo cp ./travis-scripts/fontforge.rb /usr/local/Library/Formula/fontforge.rb
```

Comment:
Using `sudo` here could potentially cause file permission problems for `brew`. If you remove the existing file first, you shouldn't need the `sudo`.